Manuscript/Sasaki_and_Moreno_2023.Rmd

```diff
@@ -27,13 +27,11 @@ output:
 
-Long term increases in ocean temperatures can accelerate the metabolisms of marine organisms, due to the fact that most are ectothermic, linking the rates of their metabolic processes to their thermal environment @johnston2022. As a result, caloric requirements will increase, increasing consumption pressure on autotrophic organisms. Modeling of community-scale metabolism reveals that increases in ocean temperature could result in increased NPP and decreases in total community biomass while community structure shifted to increased heterotrophic biomass relative to autotrophic biomass @archibald2022 . although the precise changes in community structure were heavily dependent on region, the aforementioned changes could result in greater intra- and inter- specific competition, and overall result in the top-down control of prey populations by predators. Furthermore, The reduction in dissolved oxygen associated with increased ocean temperatures further exacerbates resource limitations imposed by increased metabolic requirements and changes in marine community structure. Moreover, according to the 'evolutionary speed' hypothesis, the processes of speciation and adaptive evolution could be hastened, due to the increase in mutation rate and shorter generation times associated with increases in environmental temperature. @johnston2022. (last sentence may be tangential. thoughts?)
+Long term increases in ocean temperatures can accelerate the metabolisms of marine organisms, due to the fact that most are ectothermic, linking the rates of their metabolic processes to their thermal environment @johnston2022. As a result, caloric requirements will increase, thus increasing consumption pressure on autotrophic organisms. Modeling of community-scale metabolism reveals that increases in ocean temperatures could result in increased NPP and decreased overall community biomass, while community structure shifted to increased heterotrophic biomass relative to autotrophic biomass @archibald2022 . although the precise changes in community structure were heavily dependent on region, the aforementioned changes could result in greater intra- and inter- specific competition, and overall result in the control of prey populations by apex predators @johnston2022. Furthermore, The reduction in dissolved oxygen associated with increased ocean temperatures further exacerbates resource limitations imposed by increased metabolic requirements and changes in marine community structure. On the other hand, MHWs threaten the stability of marine ecosystems by exposing marine organisms to temperatures beyond their optimum and into the lethal range. MHWs have grown in duration, intensity, and frequency over the last 4 decades @li2022, which in tandem with the aforementioned limitation of food resources, emphasizes the importance of studying the upper thermal limits of marine organisms, as represented by Critical Thermal Maxima (CT~max~).
```

Comment:
Make sure you define any acronyms before their first use. "NPP" should be defined ahead of time. As a general rule, acronyms should only be used for especially common terms - if NPP would only be used a couple times, probably better to just write out net primary productivity instead.